### DISCUSSION TITLE
Pre-Release 1.0.3 Alpha discussion

### DISCUSSION THREAD

**Discussion by dwikurniantom** [Meruya-Techhnology/network_inspector#36] [Announcements]
Hello please gather everything that we want to be release at 1.0.3 Alpha here, and do flight check before publish to pub.dev, thank you verymuch.

**Comment by dwikurniantom** [comment]
As per 11 May 2022 we have multiple PR on going from @nEdAy that will be on 1.0.3 Alpha, thank you
#35 fix overflow rendered
#34 adding delete all action

**Comment by dwievermos** [comment]
Announcement ~ we will be releasing 1.0.3 Alpha on 20 May. All issues after the date will be picked up on the next release, Thank you

**Reply by dwikurniantom** [reply]
New PR from @MaximilianFlechtner Thank you
#38 update package depedencies to the newest

**Comment by dwikurniantom** [comment]
Sorry for the +4 day late release, there is so little time and so many things to do
